.github/workflows/ci_pipeline.yml

```diff
@@ -82,6 +87,73 @@ jobs:
             data_3.10
             coverage_${{ matrix.env }}_3.10.dat
 
+  project_cpu_tests_linux:
+    runs-on: ubuntu-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        env: 
+          - AllenCahn_Bayreuth
+          - AsympConv
+          - DAE
+          - FastWaveSlowWave
+          # - GPU
+          - Hamiltonian
+          - matrixPFASST
+          - parallelSDC
+          - parallelSDC_reloaded
+          - PinTSimE
+          - RDC
+          - Resilience
+          - SDC_showdown
+          - Second_orderSDC
+          - soft_failure
+          - TOMS
+        # python: ['3.8', '3.9', '3.10']
+        python: ['3.10']
+
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Install Conda environment with Micromamba
+        uses: mamba-org/setup-micromamba@v1
+        with:
+          environment-file: "pySDC/projects/${{ matrix.env }}/environment.yml"
+          create-args: >-
+              python=${{ matrix.python }}
+
+      - name: Install additional packages as needed
+        run: |
+          micromamba install -y --file etc/environment-tests.yml --freeze-installed
+
+      - name: Run pytest for CPU stuff
+        run: |
+          echo "print('Loading sitecustomize.py...')
+          import coverage
+          coverage.process_startup() " > sitecustomize.py
+          coverage run -m pytest --continue-on-collection-errors -v --durations=0 pySDC/projects/${{ matrix.env }}/tests
```

Comment:
Did you check that the website looks good? The plots are put into a relative path, which is likely wrong when we move the tests. Sorry that I only remember this now. Maybe we better put the tests back to where they were. Then we can use the same matrix for everything. But the matrix we can do either way, I think, when we use the pytest markers and just test everything in `pySDC/tests` and `pySDC/tests/projects`.